test/e2e/config/azure-dev.yaml

```diff
@@ -257,6 +257,7 @@ intervals:
   default/wait-control-plane-ha: ["30m", "10s"]
   default/wait-worker-nodes: ["25m", "10s"]
   default/wait-gpu-nodes: ["30m", "10s"]
+  default/wait-nodes-ready: ["10m", "10s"]
```

Comment:
This particular change relates to removing the expected etcd and coredns versions in #5528 and should fix the `Timed out after ~1s.` errors in the workload-upgrade tests. The checks that we're now skipping must have taken just enough time that this implicitly default 1s timeout before wasn't causing issues. These values are taken from CAPI.